docs/mllib-feature-extraction.md

```diff
@@ -370,3 +370,58 @@ data2 = labels.zip(normalizer2.transform(features))
 {% endhighlight %}
 </div>
 </div>
+
+## PCA
+
+PCA transform is a easy way to reduction size of vectors.
```

Comment:
Please check the grammar of the entire section. You can use the grammar checker that comes with Word or other editors.